plugins/modules/remote_management/redfish/redfish_command.py

```diff
@@ -87,6 +87,12 @@
     description:
       - BootNext target when bootdevice is "UefiBootNext"
     type: str
+  override_enabled:
+    required: false
+    description:
+      - Once, Continuous, or Disabled boot target override mode
+    default: 'Once'
+    type: str
```

Comment:
```suggestion
    type: str
    version_added: 1.2.0
```

Reply:
I'm getting an error attempting to run antsibull-changelog release:

```
ERROR! module community.general.cloud.kubevirt.kubevirt_cdi_upload missing documentation (or could not parse documentation): unknown doc_fragment(s) in file /var/folders/ty/dhjjtfbs5xg1_y941pqssphh0000gp/T/antsibull-changelogocm18a36/collections/ansible_collections/community/general/plugins/modules/cloud/kubevirt/kubevirt_cdi_upload.py: community.kubernetes.k8s_auth_options
ERROR: Uncaught exception. Run with -v to see traceback.
```
which is obviously not any module work I have been touching. Can you provide any guidance on troubleshooting?

Reply:
I don't know why you are running `antsibull-changelog release`, but in case you do, you need to install all collections that this collection depends on. It seems you have not installed `community.kubernetes`. (The full list of dependencies is here: https://github.com/ansible-collections/community.general/blob/main/galaxy.yml#L12-L15)

Reply:
Oh - I misunderstood the instructions here: https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#generating-changelogs

I will add the file to the PR and leave it at that!

Reply:
Not a problem - thanks for the quick responses! I assumed it was harder than it really was!